libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -136,7 +136,7 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @DisplayName: Maximum logging rate for file backend
     // @Description: This sets the maximum rate that streaming log messages will be logged to the file backend. A value of zero means that rate limiting is disabled.
     // @Units: Hz
-    // @Range: 0 1000
+    // @Range: 0.1 1000
```

Comment:
0.016 maybe so you can have a full min range?

Reply:
I think logging once every ten seconds is slow enough...but am open to the group's view